### ISSUE TITLE
I want to learn more about this project

### ISSUE THREAD

**Issue by jonahlibrach** [jgoldmintz/WesternBE#1]
Are you the same jgoldmintz as mentioned in this paper?

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5532611/

Would love to chat and learn more about your project.
I've built something similar, https://sciugo.com

**Comment by jgoldmintz** [comment]
Yes, that is me referenced in the paper. What questions did you have for me?
